docs/time/index.rst

```diff
@@ -854,71 +854,125 @@ Example
 
 .. EXAMPLE START: Missing Values in Time and TimeDelta Objects
 
-To set one or more items as missing, assign the special value
-`numpy.ma.masked`::
+You can set one or more items as missing when creating the object in one of two ways.
+First with a numpy masked array::
 
-  >>> t = Time(['2001:020', '2001:040', '2001:060', '2001:080'],
-  ...          out_subfmt='date')
+  >>> dates = np.ma.array(['2001:020', '...', '2001:060'], mask=[False, True, False])
+  >>> print(Time(dates, out_subfmt="date"))
+  ['2001:020'        ——— '2001:060']
+
+Second with the `astropy.utils.masked.Masked` class::
+
+  >>> from astropy.utils.masked import Masked
+  >>> dates = Masked(['2001:020', '', '2001:060'], mask=[False, True, False])
+  >>> t = Time(dates, out_subfmt="date")
+  >>> print(t)
+  ['2001:020'        ——— '2001:060']
+
+An important point is that any value which is marked as missing will be replaced
+internally with a time equivalent to ``2000-01-01 12:00:00`` UTC.
```

Comment:
The time equivalent is not in UTC, it is whatever the scale when creating the object.
```
In [2]: tm = Time(np.ma.array(["..."], mask=[True]), format="isot", scale="tai")

In [3]: tm.unmasked
Out[3]: <Time object: scale='tai' format='isot' value=['2000-01-01T12:00:00.000']>

In [4]: tm.utc.unmasked
Out[4]: <Time object: scale='utc' format='isot' value=['2000-01-01T11:59:28.000']>
```

Reply:
Oops, yes, of course, that makes sense since in the end only the value is transferred, not the scale. I adjusted it -- though as I looked at the text, I felt the whole part made more sense as part of the `.. note`. Hope you agree...